clowder-framework/clowder2-frontend
# Popup error for when unable to connect to backend #50

READY TO BE REVIEWED

This PR contains:
- delete confirmation dialog when deleting dataset and file
- error message in the login/registration populates the backend error
- error dialog displaying the response when there is any backend failure

To test:
- you can turn off the backend when frontend is at various page and see if error message pop up

**There are still a few things not working:**
- Report button need to hook up to email mailing list or something
- need to move a few functions in the "util" to action and use redux; since we need to redux to dispatch the error response - Will address that in separate task https://github.com/clowder-framework/clowder2-frontend/issues/54
~~- select dataset/ upload files not able to display messages~~

https://github.com/clowder-framework/clowder2-frontend/pull/53#issue-1110647305
![image](https://user-images.githubusercontent.com/13950475/150560807-7f55dd01-33c3-4c6e-9a4b-d1b60186613f.png)
![image](https://user-images.githubusercontent.com/13950475/150562610-086ab9cf-7008-4bf8-aa8d-efdc02fc06e6.png)
![image](https://user-images.githubusercontent.com/13950475/150562764-f2e5359c-2b92-4649-8b37-74f13788a0ec.png)
